include/libtorrent/disk_buffer_pool.hpp

```diff
@@ -96,7 +96,7 @@ namespace libtorrent {
 		}
 		int num_to_evict(int num_needed = 0);
 
-		void set_settings(aux::session_settings const& sett);
```

Comment:
I personally prefer `virtual` on methods that are overridden, otherwise it can be tricky to infer the right "polymorphic" behavior if a variable is of the base type.